scripts/agency_fix.py

```
@@ -0,0 +1,336 @@
+            )
+            resp = client.chat.completions.create(
+                model=NVIDIA_MODEL,
+                messages=messages,  # type: ignore[arg-type]
+                temperature=0.1,
+                max_tokens=8192,
+            )
+            if not resp.choices:
+                log.warning("NVIDIA NIM returned empty choices list")
+            else:
+                return resp.choices[0].message.content or ""
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)